backend/src/utils/resourceUtils.ts

```diff
@@ -429,9 +417,6 @@ const fetchDocs = async (fastify: KubeFastifyInstance): Promise<OdhDocument[]> =
         _continue = res.body?.metadata?._continue;
 
         odhDocuments.forEach((doc) => {
-          if (doc.spec.appName === 'jupyterhub' && checkJupyterEnabled()) {
```

Comment:
Not particularly wrong, but this logic is still really bad, we should improve this soon.